ChatQuickstart/QuickstartChatManager.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  QuickstartChatManager.swift
+//  ChatQuickstart
+//
+//  Created by Jeffrey Linwood on 3/11/20.
+//  Copyright © 2020 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+
+import TwilioChatClient
+
+class QuickstartChatManager:NSObject, TwilioChatClientDelegate {
+    
+    
+    // Important - update this URL with your Twilio Function URL
+    let tokenURL = "https://YOUR_TWILIO_FUNCTION_DOMAIN_HERE.twil.io/chat-token"
+    
+    var chatManagerDelegate: QuickstartChatManagerDelegate?
+    
+    // MARK: Chat variables
+    var client: TwilioChatClient? = nil
```

Comment:
Mark not available outside of this class properties and functions with `private`.